packages/@adobe/spectrum-css-temp/components/button/skin.css

```diff
@@ -1050,11 +1049,17 @@ governing permissions and limitations under the License.
     &:focus-ring,
     &.is-focused {
       &:not(.spectrum-FieldButton--quiet) {
-        outline:2px solid Highlight;
+        outline: var(--spectrum-focus-ring-size) solid var(--spectrum-fieldbutton-border-color-key-focus);
+        &:after {
+          forced-color-adjust: none;
+          box-shadow: 0 0 0 var(--spectrum-focus-ring-size) var(--spectrum-fieldbutton-border-color-key-focus);
```

Comment:
I think the `%spectrum-FocusRing` mixin will already apply `--spectrum-high-contrast-focus-ring-color` when defined. Further up this file there is a selector that defines it for Button and ActionButton. Maybe that rule could also apply to `.spectrum-FieldButton` and we could get rid of this override entirely?

Reply:
I'll look into it.